Modules/Quotas/src/Quotas.Infrastructure/Persistence/Repository/FilesRepository.cs

```diff
@@ -19,9 +20,9 @@ public async Task<uint> Count(string uploader, DateTime createdAtFrom, DateTime
         return (uint)count;
     }
 
-    public async Task<uint> UsedSpace(string uploader, DateTime from, DateTime to, CancellationToken cancellationToken)
+    public async Task<ulong> AggregateUsedSpace(string uploader, DateTime from, DateTime to, CancellationToken cancellationToken)
     {
-        var totalSpace = await _readOnlyFiles.Where(f => f.CreatedBy == uploader).SumAsync(f => (long)f.CipherSize, cancellationToken);
-        return (uint)(totalSpace / 1024 / 1024);
+        var totalSpace = await _readOnlyFiles.Where(f => f.CreatedBy == uploader).SumAsync(f => f.CipherSize, cancellationToken);
+        return (ulong)totalSpace;
```

Comment:
You should return a `long` instead of `ulong`. It's quite messy with all the types that are involved in this metric:
* long as the type of the `CipherSize` colum
* ulong as the return type of the `IFilesRepository.AggregateUsedSpace` method
* `uint` as the return type of the `IMetricCalculator.CalculateUsage` method